usage/rest-client/src/main/java/brooklyn/rest/client/BrooklynApi.java

```diff
@@ -197,6 +216,11 @@ public AccessApi getAccessApi() {
     }
 
     public static <T> T getEntity(Response response, Class<T> type) {
```

Comment:
This is called later by consumers of the api, which still returns a Response Object (which has had its contents read from one Response into another reflectively).